CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Added
+* **css + dso-toolkit + sources:** HTML/CSS Component Lists ([#1397](https://github.com/dso-toolkit/dso-toolkit/issues/1397))
```

Comment:
Dit is een change, geen toevoeging. En graag "verhuizing" toevoegen, zodat afnemers beter kunnen inschatten wat dit voor ze betekent.

En de naam van het component is **List**, niet Lists